.github/workflows/test-pythonpath-ubuntu.yml

```diff
@@ -57,10 +57,12 @@ jobs:
           # Uninstall 'isofit', leaving its dependencies.
           python3 -m pip uninstall --yes isofit
           
-          # Move 'isofit' into a new directory to ensure it is not just
+          # Move 'isofit' and copy its data directories into a new directory to ensure it is not just
           # inherently available.
           mkdir isolation/
           mv isofit/ isolation/
+          cp -r examples isolation/
```

Comment:
Why `cp` instead of `mv`?

Reply:
For some reason, we need the `examples` folder both in the `isolation/` and the originating directory. I'd need to dig up the details, cannot say for sure why right now.

Reply:
Discussed verbally offline: This will be fixed by integrating #469 into the test cases and workflows in a future PR.